zscroll

```diff
@@ -254,7 +282,7 @@ def shell_output(command):
     """
     try:
         return (
-            check_output(split(command)).decode(encoding='UTF-8').rstrip('\n')
+            check_output(command if shell else split(command), shell=shell).decode(encoding='UTF-8').rstrip('\n')
```

Comment:
I think you'll have to reformat with black. I don't have automated linting setup right now, so don't worry about this if it's not a quick change. I can fix it later.